assets/test/specs/details.spec.js

```diff
@@ -0,0 +1,119 @@
+require("jquery");
+
+describe("Tests for the <details> polyfill for pre-historic browsers.", function ()
+{
+  var details1 = "#details-1";
+  var details2 = "#details-2";
+  var selectors = details1 + ", " + details2;
+
+  beforeEach(function ()
+  {
+    jasmine.getFixtures().fixturesPath = "base/specs/fixtures/";
+
+    loadFixtures('details-polyfill-fixture.html');
+
+    require('../../javascripts/modules/details.polyfill.js');
+  });
+
+  afterEach(function ()
+  {
+    jasmine.clock().uninstall();
+  });
+
+  describe("How it works when details got initialised on page load.", function ()
+  {
+    beforeEach(function ()
+    {
+      $("details").details();
+    });
+
+    it("Should initialise details polyfill for all <details> elements.", function ()
+    {
+      $(selectors).each(function ()
+      {
+        expect($(this).prop("details-initialised")).toBeTruthy();
+      });
+    });
+
+    it("Should set content ids.", function ()
+    {
+      expect($("div", details1).attr("id")).toMatch(/details-content-\d+/);
+      expect($("div", details1).attr("id")).toMatch(/details-content-\d+/);
+    });
+
+    it("Should contain arrows.", function ()
+    {
+      expect($("summary>i.arrow.arrow-open", details1).size()).toEqual(1);
+      expect($("summary>i.arrow.arrow-closed", details1).size()).toEqual(1);
```

Comment:
could also create some local vars here as your using the same query multiple times